drivers/ethernet/eth_enc28j60.c

```diff
@@ -43,12 +50,19 @@ static void eth_enc28j60_set_bank(struct device *dev, u16_t reg_addr)
 	tx_buf[0] = ENC28J60_SPI_RCR | ENC28J60_REG_ECON1;
 	tx_buf[1] = 0x0;
 
-	spi_transceive(context->spi, tx_buf, 2, tx_buf, 2);
+	struct spi_buf tx_bufs[] = {
```

Comment:
Always declare variables at the beginning of a code block, not in the middle, so move this up.
Apply that rule everywhere.